components/board/server_test.go

```diff
@@ -15,7 +15,11 @@ import (
 	"go.viam.com/rdk/testutils/inject"
 )
 
-var errFoo = errors.New("whoops")
+var (
+	errFoo           = errors.New("whoops")
+	errUnimplemented = errors.New("not found")
+	errStatusFailed  = errors.New("no status")
```

Comment:
This variable is not used in this file, and only used in client_test.go. Mild preference to move it over there, though I don't feel strongly.

...it gets returned over there, but never checked. Does it need to exist at all?

Reply:
good point. removing